.github/component_owners.yml

```diff
@@ -0,0 +1,6 @@
+# Test configuration
+
+components:
```

Comment:
Shall we also own https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/packages/opentelemetry-id-generator-aws-xray?

Reply:
And https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-aws-lambda?

Reply:
yes